.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: release
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  release:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Extract TaxJar Version
+        run: echo ::set-env name=TAXJAR_VERSION::$(echo ${GITHUB_REF:11})
+      - name: Build release DLL for .NET Standard 2.0
+        run: |
+          sed -i 's/^\(\s*\)<TargetFrameworks>.*$/\1<TargetFrameworks>netstandard2.0<\/TargetFrameworks>/' src/Taxjar/Taxjar.csproj
+          dotnet restore src/Taxjar/Taxjar.csproj
+          msbuild src/Taxjar/Taxjar.csproj /p:configuration=Release
+          cp src/Taxjar/bin/Release/netstandard2.0/TaxJar.dll build/netstandard2.0
+      - name: Build release DLL for .NET 4.5.2
+        run: |
+          sed -i 's/^\(\s*\)<TargetFrameworks>.*$/\1<TargetFrameworks>net452<\/TargetFrameworks>/' src/Taxjar/Taxjar.csproj
+          dotnet restore src/Taxjar/Taxjar.csproj
+          msbuild src/Taxjar/Taxjar.csproj /p:configuration=Release /p:targetframework=net452
+          cp src/Taxjar/bin/Release/net452/TaxJar.dll build/net452
+      - name: Create NuGet package
+        run: |
+          sed -i 's/^\(\s*\)<TargetFrameworks>.*$/\1<TargetFrameworks>netstandard2.0;net452<\/TargetFrameworks>/' src/Taxjar/Taxjar.csproj
```

Comment:
Not sure this line is necessary, but it's included in our current release procedures.

Reply:
I'm not really sure what this is. Is this maybe adding the multi-targeting line?

Reply:
@codymullins Exactly. It's just resetting `src/Taxjar/Taxjar.csproj` to `<TargetFrameworks>netstandard2.0;net456</TargetFrameworks>`. I just wasn't sure that's strictly necessary before running the next step: `mono build/nuget.exe pack build/TaxJar.nuspec`.

Reply:
I think this is fine.